content/en/agent/kubernetes/_index.md

```diff
@@ -174,11 +174,155 @@ To install the Datadog Agent on your Kubernetes cluster:
+
+To use the Datadog Operator, deploy it in your Kubernetes cluster. Then create a `DatadogAgent` Kubernetes resource that contains the Datadog deployment configuration:
+
+1. Download the [Datadog Operator project zip ball][4]. Source code can be found at [`DataDog/datadog-operator`][1].
+2. Unzip the project, and go into the `./datadog-operator` folder.
+3. Define your namespace and operator:
+
+   ```shell
+   DD_NAMESPACE="datadog"
+   DD_NAMEOP="ddoperator"
+   ```
+
+4. Create the namespace:
+
+   ```shell
+   kubectl create ns $DD_NAMESPACE
+   ```
+
+5. Install the operator with Helm:
+
+   - Helm v2:
+
+   ```shell
+   helm install --name $DD_NAMEOP -n $DD_NAMESPACE ./chart/datadog-operator
+   ```
+
+   - Helm v3:
+
+   ```shell
+   helm install $DD_NAMEOP -n $DD_NAMESPACE ./chart/datadog-operator
+   ```
+
+## Deploy the Datadog Agents with the operator
+
+After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the Datadog Agent's deployment in your Kubernetes cluster. By creating this resource in the `Datadog-Operator` namespace, the Agent is deployed as a `DaemonSet` on every `Node` of your cluster.
+
+Create the `datadog-agent.yaml` manifest out of one of the following templates:
+
+* [Manifest with Logs, APM, process, and metrics collection enabled.][5]
+* [Manifest with Logs, APM, and metrics collection enabled.][6]
+* [Manifest with Logs and metrics collection enabled.][7]
+* [Manifest with APM and metrics collection enabled.][8]
+* [Manifest with Cluster Agent.][9]
+* [Manifest with tolerations.][10]
+
+Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your [Datadog API and application keys][11], then trigger the Agent installation with the following command:
+
+```shell
+$ kubectl apply -n $DD_NAMESPACE -f datadog-agent.yaml
+datadogagent.datadoghq.com/datadog created
+```
+
+You can check the state of the `DatadogAgent` ressource with:
+
+```shell
+kubectl get -n $DD_NAMESPACE dd datadog
+NAME            ACTIVE   AGENT             CLUSTER-AGENT   CLUSTER-CHECKS-RUNNER   AGE
+datadog-agent   True     Running (2/2/2)                                           110m
+```
+
+In a 2-worker-nodes cluster, you should see the Agent pods created on each node.
+
+```shell
+$ kubectl get -n $DD_NAMESPACE daemonset
+NAME            DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+datadog-agent   2         2         2       2            2           <none>          5m30s
+
+$ kubectl get -n $DD_NAMESPACE pod -owide
+NAME                                         READY   STATUS    RESTARTS   AGE     IP            NODE
+agent-datadog-operator-d897fc9b-7wbsf        1/1     Running   0          1h      10.244.2.11   kind-worker
+datadog-agent-k26tp                          1/1     Running   0          5m59s   10.244.2.13   kind-worker
+datadog-agent-zcxx7                          1/1     Running   0          5m59s   10.244.1.7    kind-worker2
+```
+### Tolerations
```

Comment:
This section is a bit confusing to me. I think we should move it out from this page to an advanced topic, and also explain why this is needed, and what is the configuration that the user needs to add (the current yaml snippet looks like it's replacing the entire DatadogAgent manifest that was previously downloaded, as opposed to adding the tolerations field only)